_posts/en/meetings/2016-07-07-meeting.md

```diff
@@ -0,0 +1,108 @@
+---
+title: IRC meeting summary for 2016-07-07
+permalink: /en/meetings/2016/07/07/
+name: 2016-07-07-meeting
+type: meetings
+layout: page
+lang: en
+version: 1
+---
+{% include _toc.html %}
+ 
+- [Link to this week logs](https://botbot.me/freenode/bitcoin-core-dev/2016-07-07/?msg=69273712&page=2)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-07-07-19.00.html)
+ 
+---
+ 
+## Notes / short topics
+
+Gmaxwell notes he'd still like to see separate testnet-defaulting binaries during the RC's. Wumpus points at a [PR][#8285] he missed that makes it very easy to install a testnet client.
+
+## Main topics
+ 
+- 0.13.0 rc1 & HD wallet issue
+- FIBRE announcement
+
+## 0.13.0 rc1
+
+### background
+ 
+0.13.0 is aimed to be released on 2016-08-01. ([full schedule](https://github.com/bitcoin/bitcoin/issues/7679))
```

Comment:
I would rather we said something like "We are getting ready for RC1 - the topic is about RC1, not about final release". Let's not talk about final release just yet.